habitat_baselines/il/models/models.py

```diff
@@ -284,7 +286,7 @@ def __init__(
             "checkpoint_path": eqa_cnn_pretrain_ckpt_path,
             "freeze_encoder": freeze_encoder,
         }
-        self.cnn = MultitaskCNN(**cnn_kwargs)
+        self.cnn = MultitaskCNN(**cnn_kwargs)  # type:ignore
```

Comment:
I is possible to make cnn_kwargs a typing.TypedDict but unfortunately, it is not available on all Python versions.

Reply:
I could not find an easy way to import TypedDict in a way that is compatible will Python 3.7 and Python 3.8 AND that makes mypy happy. Is it okay to leave it like this until Python 3.7 and lower are no longer supported?